docs-requirements.txt

```diff
@@ -59,6 +59,8 @@ pytz==2022.4
     # via babel
 requests==2.28.1
     # via sphinx
+setuptools-scm==7.0.5 
```

Comment:
This doesn't seem needed/related. Also, it's a bad idea to change behavior together with refactoring. If this plugin is needed, it should be integrated separately.